ts/store/reducers/entities/services/readStateByService.ts

```diff
@@ -0,0 +1,40 @@
+import { getType } from "typesafe-actions";
```

Comment:
The file name must be "readStateByServiceId"

Reply:
Fix in 6f0b05f